templates/install.tmpl

```diff
@@ -351,5 +351,4 @@
 		</div>
 	</div>
 </div>
-<img class="tw-hidden" src="{{AssetUrlPrefix}}/img/loading.png">
```

Comment:
Why this line removed?

Reply:
It is not used. I know why it was there but too complex to explain.

I have tested the "install" procedure.